VAMobile/src/api/authorizedServices/getAuthorizedServices.tsx

```diff
@@ -24,5 +24,6 @@ export const useAuthorizedServices = (options?: { enabled?: boolean }) => {
     meta: {
       errorName: 'getAuthorizedServices: Service error',
     },
+    staleTime: 300000, // 5 minutes
```

Comment:
Since we'll be calling this query in many other queries, I bumped up the stale time to 5 minutes. There's an argument to be made for increasing it even more, but we can adjust this as we see fit